smartsim/_core/_cli/build.py

```diff
@@ -214,7 +231,10 @@ def build_database(self) -> None:
         )
         if not database_builder.is_built:
             logger.info(
-                f"Building {database_name} version {self.versions.REDIS} from {self.versions.REDIS_URL}"
+                "Building {} version {} from {}",
+                database_name,
+                self.versions.REDIS,
+                self.versions.REDIS_URL,
```

Comment:
Seems like it would be better to just do a multiline string here:
```python
f"Building {database_name} version {self.versions.REDIS} "
f"from {self.versions.REDIS_URL}"
```

Reply:
used multiline string